ops/services/okta-global/main.tf

```diff
@@ -15,25 +15,31 @@ resource "okta_auth_server_scope" "sr" {
   default          = false
 }
 
-// Add the required claims
-resource "okta_auth_server_claim" "sr_user" {
```

Comment:
Were the `sr_user`, `sr_org`, and `groups` Okta claims being used by SR? I think this PR removes all of those.

Reply:
They are not (no claims are being used currently, just the app membership management).